### ISSUE TITLE
Failed to start the script

### ISSUE THREAD

**Issue by klvcm** [haxi0/Derootifier#5]
![59765B3C-C9DE-4BF5-B5D6-8FA53C3F6F8E](https://user-images.githubusercontent.com/132755314/236622498-92ec8d4e-fbbb-4e76-9416-fdb891140710.png)
![48280018-5A28-4CC1-B0FA-4B2852FEE06F](https://user-images.githubusercontent.com/132755314/236622500-5aceccae-5494-4f16-9ee9-a818f0ef3306.png)
![3EC9B697-A54E-4B38-A74B-2E0C5B7A6FAE](https://user-images.githubusercontent.com/132755314/236622501-f1f6e9e2-fd1d-4969-80c0-bf1c9f52510c.png)
![CCEF929F-6772-4A25-8707-85A24A63F9D3](https://user-images.githubusercontent.com/132755314/236622502-1c6a9554-48d4-465f-b8c6-c2ffbd223715.png)

**Comment by jebediah2** [comment]
I have the same issue even with all dependencies installed.

**Comment by salvatore8686** [comment]
Same problem

**Comment by likkai** [comment]
Same problem, I installed the ipa using appsyncunified or trollstore

**Comment by ablaman** [comment]
I had this problem until I installed odcctools

**Comment by haxi0** [comment]
people, it literally says please install file

**Comment by jebediah2** [comment]
@haxi0 we literally installed all what we're supposed to install. How is closing the issue useful?

**Comment by JEmlay** [comment]
Not a single person here has explained what device, what iOS, what jailbreak or how they installed the app.  Can't help people when they give zero information.

**Comment by jebediah2** [comment]
Everyone made the fair assumption that the message prompt was a specific symptom the dev could interpret. Dev doesn't even provide install instructions outside the first-run prompt mentioning dependencies we all installed, closes issues like we can't read and are generally dumb, then others chime in to say we can't expect the dev to solve this if we don't even mention our iOS version, like this matters at all. Why do we even care about Derootifier, really?

**Comment by JEmlay** [comment]
iOS and literally everything else I mentioned.  Good luck then!

**Comment by jebediah2** [comment]
Farewell

**Comment by jebediah2** [comment]
Dev so lazy they can't write down the dependencies in the readme.

dpkg-dev
file
fakeroot
odcctools
Ldid

So "please install file" means please install the "file" package. So obvious right?

**Comment by JEmlay** [comment]
Had you stated what you've installed that would have been obvious.  Learn how to report issues properly. Again, you have never given a single piece of information.  That's on you.

**Comment by ablaman** [comment]
![134ABE5D-65AE-449D-A3CB-7CC1665B658D](https://github.com/haxi0/Derootifier/assets/36988877/3e4ab90a-7288-485f-a5a1-8525e36c72fc)


Very easy to interpret. The time it didn’t work for me was my own fault. Responsibility works both ways…

**Comment by jebediah2** [comment]
You get this warning only once. The Procursus repo also has a file-cmds, which I personally installed instead of file. Can you see how easy it is to not screenshot that one-time only warning? Can you see how easy it is to get confused by the lazy writing in the subsequent error message? Can you see how easy it is to copy-paste the initial warning into the project's readme? The difference between the dev and us is we made honest mistakes while the dev is just proud to be lazy. 

Had I followed your prime advice I would have mentioned all kinds of info that are completely irrelevant, and as far as I can tell you didn't understand the lazy sentence "please install file" either, be it from the screenshot or the dev's answer here. So don't you guys go around teaching lessons with the dev's junk all over your face doing your best to troll us and be unuseful. It's really the opposite of being constructive.

**Comment by likkai** [comment]
> Not a single person here has explained what device, what iOS, what jailbreak or how they installed the app. Can't help people when they give zero information.

If you are not a developer and don't have the answer to this issue, just don't comment anything and keep your mouth shut. You are simply messing around here 🙂

**Comment by JEmlay** [comment]
Again people, you have to provide information for help.  Otherwise you won’t get any.  The OP posted their error and it was pointed out exactly what to do.  Also, anyone can help.  In the future it would do you well to watch your mouth on this site.

**Comment by jebediah2** [comment]
Like @likkai said, if you have no relevant input to give, then just stay away. Stop acting like you're the dev's appointed bodyguard.

**Comment by JEmlay** [comment]
I gave you input.  Provide proper information so people can help you.  Next time provide proper info.  The reason I keep replying is because you keep causing notifications in my inbox.  Am I to guess you're going to keep trolling this thread?

Like @ablaman said above.  Responsibility works both ways.  The only thing you are doing now it throwing a tantrum.

Provide proper information when you request help.

**Comment by jebediah2** [comment]
We were the people seeking help, you're the troller. The answer was already provided before you even chimed in. This means we provided enough information to get our answer, and that your retarded todo list of information has always been useless in this thread. All what we needed to do was decypher the lazy dev's answer, which we did without your help and without saying which iOS we're on.

Getting messages in your inbox is your own fault for subscribing, and doesn't mean you don't have the option to keep your mouth shut once and for all.

So to summarise nobody ever needed your input here and your input made zero difference. We solved it without you, so yeah I'm sorry if you're useless. Try elsewhere.

**Comment by JEmlay** [comment]
So to summarize, you plan to keep trolling.  Whatever floats your boat.

Hopefully next time you get your act together and actually provide proper information because to date you have done nothing but troll, cry, throw a tantrum and act like a complete child.  I legit tried to help you but then you acted like garbage.  That’s on you.  Grow up child.

**Comment by likkai** [comment]
> I legit tried to help you

Enough with your "help", now we've all got our answer and it's time for you to unsubscribe to this issue and "help" others. In case you didn't know, you @JEmlay , helped nobody here and maybe next time you can learn to comment politely. Learn some manners, child

**Comment by JEmlay** [comment]
Sure, as soon as you two learn how to provide proper information and be polite yourselves.  Or is this what you consider polite,  "keep your mouth shut"?  Both of you need to grow up and be respectful to both devs and people who try to help you.  The two of you endlessly replying with the same drivel is ridiculous.  Imagine being so childish that you fight the fact you should properly post information about your issue.  I can't even image being that childish.

**Comment by likkai** [comment]
> Had you stated what you've installed that would have been obvious. Learn how to report issues properly. Again, you have never given a single piece of information. That's on you.

How polite, I bet the word polite isn't even in your dictionary. If you don't know the answer to this issue, just ignore it instead of commenting something not helpful after the issue has been solved. You don't have to prove your existence on GitHub by commenting something not helpful on someone else's issue. 

Just stop replying to this issue, you don't have this issue and you are irrelevant here. @JEmlay

**Comment by JEmlay** [comment]
> > Had you stated what you've installed that would have been obvious. Learn how to report issues properly. Again, you have never given a single piece of information. That's on you.
> 
> How polite, I bet the word polite isn't even in your dictionary. If you don't know the answer to this issue, just ignore it instead of commenting something not helpful after the issue has been solved. You don't have to prove your existence on GitHub by commenting something not helpful on someone else's issue.
> 
> Just stop replying to this issue, you don't have this issue and you are irrelevant here. @JEmlay

Learn to provide proper information when asking for help and learn to be polite.

**Comment by likkai** [comment]
> Learn to provide proper information when asking for help and learn to be polite.

By a completely irrelevant, unhelpful and impolite kid. (Not even a Dev, just a busybody)

**Comment by JEmlay** [comment]
> > Learn to provide proper information when asking for help and learn to be polite.
> 
> By a completely irrelevant, unhelpful and impolite kid. (Not even a Dev, just a busybody)

Cool story.  Hopefully you learned something here about how to properly ask for help.
